barretenberg/cpp/src/barretenberg/commitment_schemes/commit.bench.cpp

```diff
@@ -7,25 +7,137 @@ namespace bb {
 
```

Comment:
This suite was previously just benchmarking committing to zero polynomials of various sizes. I'm assuming it was just a WiP and never used but who knows. I've updated it to include a number of different scenarios, including committing various types of sparse polynomials with the traditional `commit()` and the new `commit_sparse()`